### PR TITLE
7-vote-on-article-refactor

### PR DESCRIPTION
**Pull request by euandecoup**

### CODE REVIEW

File: src/Components/CommentsList.jsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import React, { useEffect, useState } from 'react'
 import { fetchData } from '../utils/api'
-import { useParams } from 'react-router-dom'
+import { useParams, useSearchParams } from 'react-router-dom'
+import CommentAdder from './CommentAdder'
```
Comment (nc-hannah-burks) [inline src/Components/CommentsList.jsx:4]: - Getting a build error due to ./CommentAdder being referenced in CommentList.jsx but no longer existing?

**Review by nc-hannah-burks — COMMENTED** [review]
Your individual article pages are currently crashing. No content coming through. Please try resolving this issue and then resubmit :)

**Review by nc-hannah-burks — COMMENTED** [review]
It is breaking because you haven't pushed up your comment adder component :)

**Comment by euandecoup** [comment]
Oops! That was me trying to be clever and seperate out the tickets! I've completed the next one now so will push that up with a wee message to explain :)

**Pull request merged**